src/editor-service.ts

```diff
@@ -13,8 +13,8 @@
 
 import {SourcePosition, SourceRange, Warning} from 'polymer-analyzer';
 
-export type TypeaheadCompletion =
-    ElementCompletion | AttributesCompletion | AttributeValuesCompletion;
+export type TypeaheadCompletion = ElementCompletion | AttributesCompletion |
+    AttributeValuesCompletion | DatabindingPropertiesCompletion;
 export interface ElementCompletion {
   kind: 'element-tags';
```

Comment:
The naming in this file has been kinda slapdash because it's deprecated so we haven't worked to keep it super legible. Fixing it would involve releasing a new major version as it would be a breaking change to the atom plugin, and I'd really rather just delete this file for the next breaking change.

It looks like https://github.com/atom/atom-languageclient is coming along pretty well, so that will hopefully be very soon.

I added a deprecation notice to the top of the file. WDYT?